src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java

```diff
@@ -0,0 +1,138 @@
+import co.aikar.commands.annotation.Syntax;
+import com.dreamhorizon.core.commands.implementation.DHCommand;
+import com.dreamhorizon.core.configuration.enums.Message;
+import com.dreamhorizon.core.helper.MessageHelper;
+import com.dreamhorizon.core.modulation.ModuleHandler;
+import com.dreamhorizon.core.modulation.implementation.ModuleEntry;
+import com.dreamhorizon.core.util.PaginationUtil;
+import org.bukkit.command.CommandSender;
+
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * @author Lukas Mansour
+ * @since 1.0
+ */
+@SuppressWarnings("unused")
+@CommandAlias("module|modules|dhmodule|dhmodules")
+@Description("Module administration command")
+@CommandPermission("dhcore.command.modules")
+public class ModuleCommand extends DHCommand {
+    
+    @Subcommand("list")
+    @Description("Gets a list of enabled modules")
+    @Syntax("(page)")
+    @CommandPermission("dhcore.modules.list")
+    public static void onModuleList(CommandSender sender, @Default(value = "1") int page) {
+        if (page == 0) page = 1;
+        List<ModuleEntry> paginatedModules = PaginationUtil.getPage(ModuleHandler.getInstance().getModuleEntries(), page, 8);
+        int maxPage = (int) Math.ceil(paginatedModules.size() / 8.0);
+        if (maxPage == 0) maxPage = 1;
+        if (page > maxPage) page = maxPage;
+        
+        HashMap<String, Object> placeholders = new HashMap<>();
+        placeholders.put("page", page);
+        placeholders.put("maxPage", maxPage);
+        
+        sender.sendMessage(MessageHelper.formatMessage(placeholders, Message.MODULE_LIST_HEADER));
+        for (ModuleEntry paginatedModule : paginatedModules) {
+            placeholders.put("moduleName", paginatedModule.getName());
+            placeholders.put("moduleAuthor", paginatedModule.getAuthor());
+            if (paginatedModule.getModule().isEnabled()) {
+                placeholders.put("moduleEnabled", Message.YES);
+            } else {
+                placeholders.put("moduleEnabled", Message.NO);
+            }
+            sender.sendMessage(MessageHelper.formatMessage(placeholders, Message.MODULE_LIST_ELEMENT));
+        }
+        sender.sendMessage(MessageHelper.formatMessage(placeholders, Message.MODULE_LIST_FOOTER));
+    }
+    
+    @Subcommand("info")
+    @Description("Shows info about a module")
+    @Syntax("[module]")
+    @CommandCompletion("@modules")
+    @CommandPermission("dhcore.modules.info")
+    public static void onModuleInfo(CommandSender sender, ModuleEntry moduleEntry) {
+        HashMap<String, Object> placeholders = new HashMap<>();
+        placeholders.put("moduleName", moduleEntry.getName());
+        placeholders.put("moduleAuthor", moduleEntry.getAuthor());
+        placeholders.put("moduleCommandAmount", moduleEntry.getModule().getCommands().size());
+        placeholders.put("moduleListenerAmount", moduleEntry.getModule().getListeners().size());
+        if (moduleEntry.getModule().isEnabled()) {
```

Comment:
Perhaps a ternary operator here? Not needed, just a thought.

Reply:
I hate ternary operations. just a heads up :>